core/integration/client_tests.go

```diff
@@ -65,7 +65,9 @@ var (
 
 // TestEmptyGetAndUpdate verifies set/get semantics.
 func TestEmptyGetAndUpdate(ctx context.Context, env *Env, t *testing.T) {
-	signature.RegisterStandardKeyTypes()
+	if _, err := signature.RegisterStandardKeyTypes(); err != nil {
+		t.Errorf("RegisterStandaredKeyTypes(): %v", err)
```

Comment:
Typo in the logged function name - remove the 'e' from 'Standared'

Reply:
fixed